File: python/7 kyu/317.py
# https://www.codewars.com/kata/5a5bef7a5c770d08cd0032fa/train/python

# Passed

def agents(list_found, list_records):
    result = "No matches"
    if list_found:
        for rec in list_records:
            if rec == list_found:
                result = "Match found"
                break
    else:
        result = None
            
    return result

output = agents(["Sally", "David", "Jack", "Rebeccah"], [["Sally", "David", "Jack", "Rebeccah"], ["Jennie", "Jill", "Frank"]])
print(output)

File: nodejs/7 kyu/317.js
// https://www.codewars.com/kata/582f52208278c6be55000067/train/javascript

// Passed

function roundToFive(numbers) {
  let roundedNumbers = numbers.map((num) => 5 * Math.round(num / 5));
  return roundedNumbers;
}

const output = roundToFive([34.5, 56.2, 11, 13]);
console.log(output);